src/main/java/org/camunda/community/benchmarks/ProcessDeployer.java

```diff
@@ -68,4 +74,13 @@ private InputStream adjustInputStreamBasedOnConfig(InputStream is) throws IOExce
 
         return new ByteArrayInputStream(fileContent.getBytes());
     }
+
+    @Override
+    public void onApplicationEvent(ContextRefreshedEvent event) {
```

Comment:
should this be synchronized? or does spring guarantee to call this strictly sequentially/ on a single thread/ whatever?

Reply:
springs application events are by default synchronized unless we put @Async in this method, so should be in single thread (I don't know about current state of spring tbh)